util/src/main/resources/SummaryReport.ftl

```diff
@@ -189,4 +195,15 @@
       </div>
     </#if>
   </body>
+  <script>
+    $(document).ready(function () {
+      $('#statvars-table').DataTable({
+        paging: false,
+        searching: false,
+        info: false,
+        order: [] // don't apply initial ordering (which is turned on by default)
```

Comment:
curious why we're adding this new DataTable if ordering is not being applied? What is the goal of using DataTable?

Reply:
the addition of DataTable allows users to click on columns to sort the table according to their interests.

without specifying an empty `order` configuration, DataTables will default to sorting by the first column. This alphabetically sorts the table based on StatVar names, which does not make much sense.

this line removes that default sorting, and instead displays the statvars in the order they are present in the source HTML

if we want a default sort order, we can specify that in this line 204 that you've highlighted

Reply:
ohh, I see, I couldn't see it when I opened the html in the github html previewer so I couldn't tell what the change was but the code sandbox was useful!